### PR TITLE
Refactoring code

### PR DESCRIPTION
**Pull request by NicholasLYang**
This are simple changes just to conform to JS conventions.

- Component file names should be capitalized.
- If there's a function that returns JSX, it's a component. Make it one.
- Use env vars for stuff that changes (API URLs for instance).

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
   "scripts": {
     "build:css": "postcss src/tailwind.css -o src/index.css",
     "watch:css": "postcss src/tailwind.css -o src/index.css",
-    "start": "yarn watch:css && react-scripts start",
-    "build": "yarn build:css && react-scripts build",
+    "start": "npm run watch:css && react-scripts start",
+    "build": "npm run build:css && react-scripts build",
```
Comment (A1Liu) [inline frontend/package.json:27]: We using yarn my dude

**Review by A1Liu — CHANGES_REQUESTED** [review]
Fix the NPM stuff and we can merge this

**Comment by esilverm** [comment]
imagine using npm 😤 

otherwise looks good

File: frontend/package.json (hunk 1 above)
Comment (NicholasLYang) [reply frontend/package.json:27]: The difference is that this works whether you use yarn or npm, while the prior one only works if you use yarn

**Comment by NicholasLYang** [comment]
I don't get the purpose of yarn. It's not better in any real way but it's another bit of software to download.

**Comment by A1Liu** [comment]
I find download speeds to be far faster on Yarn, and also I find that PR's are more likely to be merged when they follow the project conventions.

**Pull request closed without merging**